File: test/resources/repos/php/test_repo/ChildClass.php
<?php

require_once 'BaseClass.php';

class ChildClass extends BaseClass 
{
    private int $value;

    public function __construct(string $name, int $value) 
    {
        parent::__construct($name);
        $this->value = $value;
    }

    public function execute(): void 
    {
        echo "Executing {$this->name} with value {$this->value}\n";
    }

    public function getValue(): int 
    {
        return $this->value;
    }
}

File: test/resources/repos/csharp/test_repo/ChildClass.cs
using System;

namespace TestRepo
{
    public class ChildClass : BaseClass
    {
        private int value;

        public ChildClass(string name, int value) : base(name)
        {
            this.value = value;
        }

        public override void Execute()
        {
            Console.WriteLine($"Executing {name} with value {value}");
        }

        public int GetValue()
        {
            return value;
        }
    }
}